data/modules/TradeShips/Debug.lua

```diff
@@ -0,0 +1,333 @@
+-- Copyright © 2008-2021 Pioneer Developers. See AUTHORS.txt for details
+-- Licensed under the terms of the GPL v3. See licenses/GPL-3.txt
+
+local ShipDef = require 'ShipDef'
+local debugView = require 'pigui.views.debug'
+local Engine = require 'Engine'
+local e = require 'Equipment'
+local Game = require 'Game'
+local ui = require 'pigui.baseui'
+local utils = require 'utils'
+local Vector2 = _G.Vector2
+
+local Core = require 'modules.TradeShips.Core'
+
+local drawArrayTable = require 'pigui.libs.array-table'.draw
+local addKeys = require 'pigui.libs.array-table'.addKeys
```

Comment:
I would recommend doing `local arrayTable = require 'pigui.libs.array-table'` and then calling these functions via `arrayTable.draw()` or `arrayTable.addKeys()`; not only does it fit a bit better with existing conventions, but it also logically follows the model of using the local variable as a package namespace.

Reply:
Done!